website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -89,25 +135,27 @@ metrics:
 
     meta: {team: Finance}
 ```
+</VersionBlock> 
+
+</File>   
 
-</File>
 
 ### Available properties
 
 | Field       | Description                                                 | Example                         | Required? |
 |-------------|-------------------------------------------------------------|---------------------------------|-----------|
 | name        | A unique identifier for the metric                          | new_customers                   | yes       |
-| model       | The dbt model that powers this metric                       | dim_customers                   | <VersionBlock firstVersion="1.2">yes (no for `expression` metrics)</VersionBlock><VersionBlock lastVersion="1.1">yes</VersionBlock> |
+| model       | The dbt model that powers this metric                       | dim_customers                   | <VersionBlock firstVersion="1.3">yes (no for `derived` metrics)</VersionBlock><VersionBlock lastVersion="1.2">yes (no for `expression` metrics)</VersionBlock><VersionBlock lastVersion="1.1">yes</VersionBlock> |
 | label       | A short for name / label for the metric                     | New Customers                   | no        |
 | description | Long form, human-readable description for the metric        | The number of customers who.... | no        |
-| type        | The type of calculation to perform when evaluating a metric | count_distinct                  | yes       |
-| sql         | The expression to aggregate/calculate over                  | user_id                         | yes       |
+| <VersionBlock firstVersion="1.3">calculation_method</VersionBlock><VersionBlock lastVersion="1.1">type </VersionBlock> | <VersionBlock firstVersion="1.3">The method of calculation (aggregation or derived) that is applied to the expression</VersionBlock><VersionBlock lastVersion="1.1">The type of calculation to perform when evaluating a metric</VersionBlock>  | count_distinct | yes       |
```

Comment:
the table seems to be formatting oddly when switching versions:

**v1.2 rows under description are empty and the last row only says Not yet available - added in v1.3:**

![Screenshot 2022-09-09 at 10 07 57](https://user-images.githubusercontent.com/89008547/189315272-83ac99b2-94b9-4db2-90cf-5acfb243751c.png)

**v1.1 -> `models` row** has a trailing _yes:_  
![Screenshot 2022-09-09 at 10 11 04](https://user-images.githubusercontent.com/89008547/189315280-69256d94-de76-4f2d-89ec-79c8fa7c6453.png)